src/components/AlbumSongsTable/AlbumSongsTable.js

```diff
@@ -0,0 +1,110 @@
+import albumSongsTable from "./AlbumSongsTable.html";
+import { MusicService } from "../../services/MusicService";
+import { SongsTableComponent } from "../SongsTable/SongsTable";
+
+export class AlbumsSongTableComponent {
```

Comment:
please rename it to AlbumSongTableComponent (not Albums) to match all other names

Reply:
Corrected